### DISCUSSION TITLE
numbered list rendering > 9 ad `10mytask`

### DISCUSSION THREAD

**Discussion by molleweide** [nvim-neorg/neorg#319] [Ideas]
I noticed that a numbered list with `~` = 10 renders like this if there is just one whitespace between `~` and the following text.

```
9 sometask
10anothertask.
```

Would it make sense to auto insert an additional whitespace here in order to get it to render this: `10 anothertask`
and if so where is this implemented because I would like to try to add this feature myself.

**Comment by vhyrro** [comment]
The reason we don't do this is because it's physically impossible currently in Neovim :P

We're waiting for anticonceal to be implemented in core which will then allow this functionality, but until then we might just limit the maximum count to 9 or something

**Reply by molleweide** [reply]
allright!

naah I'm fine with inserting manually for now then ;)
